packages/ui/package.json

```diff
@@ -25,7 +25,11 @@
     "typecheck": "tsc --noEmit --project ./tsconfig.json"
   },
   "exports": {
-    ".": "./dist-web/web.js",
+    "./web": {
+      "types": "./web.ts",
```

Comment:
I'm not sure if this is right for `types` but it works. Maybe I need to package and move them to `dist` also.

Right now we export and consume some types for extending with:
- ButtonProps
- FlagProps
- TagProps
- TagVariants
- HrProps

This is a temporary step until I move the remaining components so maybe we can ignore for now. 

The types in `index.d.ts` have no exports so if I instead include that it fails to install